release/pipelines/windows-customize/windows-customize.yaml

```diff
@@ -187,6 +212,25 @@ spec:
             value: cleanup-vm
           - name: version
             value: v0.20.0
+    - name: delete-imported-configmaps
```

Comment:
Does it make sense to delete the ConfigMaps every time?

Reply:
Yes, because otherwise we will get bugs, that pipeline is not cleaning after itself :D

Reply:
ACK